### ISSUE TITLE
"Relative publication volume" has scientific notation on the y axis

### ISSUE THREAD

**Issue by janschulz** [aknutas/nails#13]
See image:

![image](https://cloud.githubusercontent.com/assets/890156/24830764/8e8f6d4c-1c8c-11e7-91fe-bf42a9dfc5ba.png)

It would be nice to have some more human readable form...

**Comment by aknutas** [comment]
You are correct that it would be much more readable this way. @jsalminen knows the graph formatting features and we will take this issue underway.
